shared/desktop/app/index.js

```diff
@@ -26,6 +27,10 @@ function start() {
       if (mainWindow) {
         mainWindow.show(true)
       }
+      if (isWindows) {
+        // Wasn't showing on Windows without this
+        showMainWindow()
```

Comment:
This also doesn't make sense because it uses ipcRenderer, but this is being called from the main process.

I would check what mainWindow is?  maybe check what `mainWindow.isVisible()` is?